accounts/identity.go

```diff
@@ -157,12 +149,12 @@ func parseKeystoreUrl(path string) (string, error) {
 	return u.Path, nil
 }
 
-// return default keystore directory stored in in `.sonm` directory
-// if any error occurred .sonm directory will be in working dir
+// GetDefaultKeystoreDir return default keystore directory places in "~/.sonm/keystore".
+// If any error occurred ".sonm" directory will be in current working dir
 func GetDefaultKeystoreDir() string {
-	rootDir, err := util.GetUserHomeDir()
+	homeDir, err := util.GetUserHomeDir()
```

Comment:
I'd rather fail completely in this case instead of magically creating `.sonm` directory whenever an user happened to run this code.

Reply:
Looks like this function used only for testing and into the examples. 

@Sokel is it OK to move this code somewhere into the `blockchain/example/`?